src/drivers/imu/invensense/icm42688p/InvenSense_ICM42688P_registers.hpp

```diff
@@ -159,6 +159,16 @@ enum SIGNAL_PATH_RESET_BIT : uint8_t {
 	FIFO_FLUSH      = Bit1,
 };
 
+// INTF_CONFIG0
+enum INTF_CONFIG0_BIT : uint8_t {
+	FIFO_HOLD_LAST_DATA_EN = Bit7,
+	FIFO_COUNT_REC         = Bit6,
+	FIFO_COUNT_ENDIAN      = Bit5,
+	SENSOR_DATA_ENDIAN     = Bit4,
+	UI_SIFS_CFG            = Bit1 | Bit0,
```

Comment:
How about something like this? We need a better register abstraction...

```suggestion
	UI_SIFS_CFG_DISABLE_I2C            = Bit1 | Bit0,
```

Reply:
Sure!

Reply:
I also added the definitions to the copy of the file under boards/modalai/voxl2-slpi/src/drivers/icm42688p just in case. I am not sure how it is used, but I guess it doesn't harm at least, as it didn't exist before.